torch_xla/csrc/aten_autograd_ops.cpp

```diff
@@ -18,6 +18,42 @@ bool IsNonTrivialDilation(at::IntArrayRef dilation) {
 
 namespace aten_autograd_ops {
 
+torch::Tensor EinsumAutogradFunction::forward(
+    torch::autograd::AutogradContext* ctx, const c10::string_view equation,
+    at::TensorList tensors) {
+  ctx->saved_data["equation"] = equation;
+
+  torch::autograd::variable_list vars;
+  for (auto const& tensor : tensors) {
+    vars.push_back(tensor);
+  }
+  ctx->save_for_backward(vars);
+
+  auto xla_tensors = bridge::GetXlaTensors(absl::MakeSpan(tensors));
```

Comment:
Let's write out the actual type here, using `auto` often make code reading a lot more difficult. same for other `autos` in this file

Reply:
Done